gitter.py

```diff
@@ -190,7 +191,11 @@ def leave(self, reason=None):
 
     @property
     def topic(self):
-        return "TODO"  # TODO
+        json = self._backend.readAPIRequest('rooms', {"q": self.uri}).json()
+        for element in json:
+            if element.uri == self.uri:
+                return element.topic
+        return "Couldn't find :("
```

Comment:
This code should raise an exception if the URI isn't present in the response, as it's supposed to be there.